Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -319,7 +319,7 @@ class PlaceDetailsCardView: UIView {
             return
         }
 
-        guard let (primaryText, secondaryText) = getStringsForOpenHours(hours, forDate: Date()) else {
+        guard let (primaryText, secondaryText) = getStringsForOpenHours(hours, forDate: Date(), isEvent: place.customProvider?.description != nil) else {
```

Comment:
It's slightly safer to additionally check: `!descrption.isEmpty`, just in case we screw up the schema.